site/pages/docs/components/custom/thumbnail.vue

```diff
@@ -0,0 +1,167 @@
+<template>
+  <div class="docs__section">
+    <div class="docs__header">
+      <h1>Thumbnail</h1>
+      <o-label size="small" text="HTML" />
+    </div>
+
+    <docs-item class="explanation">
+      <div slot="body">
+        <p>Thumbnail is a styled image which can be combined to gallery.</p>
+      </div>
+    </docs-item>
+
+    <!-- Basic -->
+    <docs-item title="Basic" :code="thumbnail_basic" v-if="is_html">
+      <div slot="body">
+        <img class="thumbnail" src="/images/thumbnails/cat.jpg" alt="Katze" />
+      </div>
+    </docs-item>
+    
+    <!-- With shadow  -->
+    <docs-item title="With shadow" :code="thumbnail_shadow" v-if="is_html">
+      <div slot="body">
+        <img class="thumbnail thumbnail--shadow" src="/images/thumbnails/hedgehog.jpg" alt="Hedgy" />
+      </div>
+    </docs-item>
+
+    <!-- In circle  -->
+    <docs-item title="In circle" :code="thumbnail_circle" v-if="is_html">
```

Comment:
this style should be able to deal with non-square images

Reply:
we can use this property to achieve this @matevzgolavsek 
https://caniuse.com/#feat=object-fit